projects/components/src/form-field/form-field.component.ts

```diff
@@ -23,7 +23,7 @@ import { IconSize } from '../icon/icon-size';
         class="content"
         [ngClass]="{
           'show-border': this.showBorder,
-          'error-border': this.showFormError && this.errorLabel
+          'error-border': this.showBorder && this.showFormError && this.errorLabel
```

Comment:
We don't need both.

Reply:
So I want to show form error label but not error-border (because it does not look good for datetime picker), since `this.showFormError` is also used for error label I cannot use that

Reply:
This could have some side effects

Reply:
Why does it look bad? If double borders are showing up then we need to remove border from date-picker through an input option